doc/source/starting.rst

```diff
@@ -0,0 +1,112 @@
+Getting Started
+===============
+
+Installation
+-----------
+The Distributions package is available through the Julia package system by running ``Pkg.add("Distributions")``. Throughout, we assume that you have installed the package.
+
+Starting With a Normal Distribution
+-----------
+
+We start by drawing :math:`100` observations from a standard-normal random variable. After setting up the environment through
+
+.. code-block:: julia
+
+    julia> using Distributions
+    julia> srand(123) # Setting the seed
+
+We first create a ``normal`` distribution and then obtain samples using ``rand``:
+
+.. code-block:: julia
+
+    julia> normal = Normal()
```

Comment:
I think using case only to distinguish an object from a type is a little confusing for a beginner.